gratipay/wireup.py

```diff
@@ -59,6 +59,11 @@ def db(env):
 
     return db
 
+def settings(db):
+    from gratipay.billing.exchanges import MINIMUM_CHARGE
+    db.run('DELETE FROM settings')
```

Comment:
> Seems a little funky to have multiple web app processes potentially running this code at the same time.

Nice catch - we could end up with two inserts, and all our `db.one` calls will fail.